.packit.yaml

```diff
@@ -40,6 +40,36 @@ jobs:
     owner: rhcontainerbot
     project: podman-next
 
+  # All tests specified in the `/plans/` subdir
+  # Podman e2e tests for Fedora and CentOS Stream
+  #- job: tests
+  #  trigger: pull_request
+  #    #notifications:
+      #failure_comment:
+      #  message: "podman e2e tests failed. @containers/packit-build please check."
+      # targets: &pr_test_targets
+      #- fedora-all-x86_64
+      #- fedora-all-aarch64
+      #- epel-9-x86_64
+      #- epel-9-aarch64
+      # identifier: podman_e2e_test
+      #tmt_plan: "/plans/podman_e2e_test"
```

Comment:
Debugging holdover? Or copy-paste? If necessary to keep this, please document why. Otherwise remove.

Reply:
yes, will be fixed before merge.